### PR TITLE
Feature/component info restrict children test

### PR DESCRIPTION
**Pull request by ruudjungbacker**
For now this branch is based on the `feature/component-info` branch but I wanted to separate. A new test component set has been added that allows to test the restrict children path for the `generateComponentSetInfo` function.

### CODE REVIEW

File: test/resources/restrict-children-sample/localization/csCZ.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (wwsno) [inline test/resources/restrict-children-sample/localization/csCZ.json:1]: You can leave out the localisation files for other languages than english in the test resources.

Comment (ruudjungbacker) [reply test/resources/restrict-children-sample/localization/csCZ.json]: Just removed them.

File: test/definition/component-info.test.ts [hunk 2]
```diff
@@ -4,65 +4,148 @@ import { generateComponentSetInfo, processInfo } from '../../lib/definition/comp
...
+                                    type: 'editable',
+                                },
+                            ],
+                        },
+                    },
+                });
+            });
+        });
+
+        describe('processInfo', () => {
```
Comment (wwsno) [inline test/definition/component-info.test.ts:61]: Another processInfo describe block? That does not make sense. You should remove one.

**Review by wwsno — APPROVED** [review]
Looks ok.

**Review by jaapvanblaaderen — APPROVED** [review]
Looks good to me. Maybe also extend the sample in the other PR with a restrictChildren property (container for example)?

File: test/definition/component-info.test.ts (hunk 2 above)
Comment (ruudjungbacker) [reply test/definition/component-info.test.ts]: Good point. Missed that one during a merge. Fixed it.

**Comment by ruudjungbacker** [comment]
> Maybe also extend the sample in the other PR with a restrictChildren property (container for example)?

As far as I know this PR does so? Or am I missing an area?

**Pull request merged**